is_core/generic_views/__init__.py

```diff
@@ -23,48 +36,28 @@ def dispatch(self, request, *args, **kwargs):
             handler = self.http_method_not_allowed
         return handler(request, *args, **kwargs)
 
-    def __getattr__(self, name):
-        for regex, method in (
-                (r'_check_(\w+)_permission', self._check_permission),
-                (r'can_call_(\w+)', self._check_call)):
-            m = re.match(regex, name)
-            if m:
-                def _call(*args, **kwargs):
-                    return method(m.group(1), *args, **kwargs)
-                return _call
-        raise AttributeError("%r object has no attribute %r" % (self.__class__, name))
-
-    def _check_call(self, name, *args, **kwargs):
-        if not hasattr(self, 'has_%s_permission' % name):
-            if settings.DEBUG:
-                raise NotImplementedError('Please implement method has_%s_permission to %s' % (name, self.__class__))
-            else:
-                return False
-
+    def _check_call(self, name, **kwargs):
```

Comment:
Trochu zavádějící název metody.

Reply:
nápady

volá se to např v  can_call_get

Reply:
ano přejmenováno na has_permission